src/App.vue

```diff
@@ -60,6 +53,10 @@ export default {
     if (this.girderRest.user) {
       this.fetchFolder();
     }
+    this.girderRest.$on(['login', 'register'], () => {
```

Comment:
The `RestClient` actually emits the `login` event on successful registration, so that's the only one you need to listen to.